packages/web-app-files/src/views/spaces/Projects.vue

```diff
@@ -6,20 +6,21 @@
       ref="createNewSpaceButton"
       key="new-space-menu-btn-enabled"
       :aria-label="$gettext('Create a new space')"
-      variation="primary"
-      appearance="filled"
-      class="oc-mb-l"
+      variation="inverse"
+      class="oc-mb-l oc-background-primary-gradient"
       data-testid="spaces-list-create-space-btn"
       @click="showCreateSpaceModal"
     >
       <oc-icon name="add" />
       <translate>Create Space</translate>
     </oc-button>
-    <span class="oc-display-block oc-mb-l">
-      <span v-text="$gettext('Access all project related files in one place.')" />
-      <a href="#" v-text="$gettext('Learn more about spaces.')" />
+    <span class="oc-display-block oc-mb-xl">
```

Comment:
Why don't you use a `div`? Converting a `span` to `display: block` indicates to me that you want to have a div. 😅